website/blog/2021-02-19-tungsten.md

```diff
@@ -0,0 +1,456 @@
+types. Do report any issue you encounter, it will help us greatly to polish the
+new feature.
+
+The feature will currently work for clients that allow virtual text to be
+inserted into the editor window, which is currently Visual Studio Code, Emacs
+and Sublime Text (after merging the
+[PR](https://github.com/scalameta/metals-sublime/pull/36)).
+
+## Configuration option for default Scala version.
+
+Previously, for any standalone files Metals would use the default Scala version
+of 2.12.12. This changed in `v0.10.0` and can either be defined with a
+setting:
+
+```json
+"metals.fallbackScalaVersion" : "3.0.0-RC1"
+```
+
+or if the setting is empty or set to `automatic`, Metals will infer the Scala
+version automatically from the project, which means we will try to choose the
+newest Scala version that is used in the project.
+
+## Code lens for copy worksheet command.
+
+In the previous version `0.9.8` a new command was added to Metals that allows
+clients to export the worksheet output for the users to share. This however was
+not easy to discover for a large chunk of the users, which is why the command is
+also now available by clicking the code lens at the top of a worksheet file.
+This will work for any clients that supports the `copy-worksheet-output`
+command.
+
+## [Scala 3] Filtering and sorting for completions.
+
+In previous versions of Metals, the completions offered would just be a list of
+what is available in the current scope. This however did not provide the best
+possible experience to the users. Thanks to great work by
+[@sellophane](https://github.com/sellophane) those initial results are now
+filtered and sorted so that the most useful methods and values will be available
+first.
+
+## [Scala 3] Basic shorter signature support for hover.
+
+Together with better completions we also included a better way to display types
+on hover. Previously, we would always display the full name with the package,
+which would mean that some methods could look pretty complicated for example:
+
+`def method(first: alpha.beta.gamma.DType, second: alpha.beta.gamma.DType)`
+
+Thanks to [@tanishiking](https://github.com/tanishiking) we no longer display
+the long type for anything that is already imported into scope. So the above
+method would look like:
+
+`def method(first: DType, second: DType)`
+
+This needs to be still improved by shortening some of the symbols that are not
+imported into scope and we should apply the same optimization in other places
+where we display the type.
+
+## Run/test capabilities when using sbt via BSP.
+
+Thanks to the great work of [@adpi2](https://github.com/adpi2) and
+[@er1c](https://github.com/er1c) on the
+[scala-debug-adapter](https://github.com/scalacenter/scala-debug-adapter) Metals
+now support running and debugging when using sbt BSP. The experience should be
+exactly the same as when using Bloop, since the new library reuses already
```

Comment:
```suggestion
almost the same as when using Bloop, since the new library reuses already
```
There are some differences because of the sbt part, for instance https://github.com/scalacenter/scala-debug-adapter/issues/32

Reply:
Fixed